protocol/src/main/protobuf/User.proto

```diff
@@ -27,6 +27,7 @@ enum RpcType {
   RESP_FUNCTION_LIST = 9;
 
   QUERY_RESULT = 10; // drillbit is reporting a query status change, most likely a terminal message, to the user
+  QUERY_WARNING = 12; // drillbit is sending a query warning to the user
```

Comment:
Warnings are part of the QueryResult object (QUERY_RESULT RpcType). So why is this type required?